src/components/Home/ListOfGames/index.tsx

```diff
@@ -10,42 +9,91 @@ import {
   Title,
 } from "./styles";
 
-const ListOfGames = () => {
-  const url = "https://public.connectnow.org.uk/applicant-test/";
-
-  const data: {
-    id: number;
-    first_relase_date: number;
-    name: string;
-    rating: number;
-    summary: string;
-  }[] = useAsyncFetch(url);
-
+// You might want to move this to a config or env variables.
+const URL = "https://public.connectnow.org.uk/applicant-test/";
 
+const ListOfGames = () => {
+  const gamesList = useGamesList();
 
   return (
     <DataWraper>
-      {data.map((game) => {
-        const rate: number = Math.floor(game?.rating / 10);
-        const date: any = game?.first_relase_date
-        console.log(date)
-
-        return (
-          <GameArticle key={game.id}>
-            <Image></Image>
-            <TextWrapper>
-              <Title>{game?.name}</Title>
-              <Paragraph>{game?.first_relase_date}</Paragraph>
-              <Paragraph>
-                {(game?.summary).split(" ").splice(0, 35).join(" ")}
-              </Paragraph>
-              <Score>{rate}</Score>
-            </TextWrapper>
-          </GameArticle>
-        );
-      })}
+      { gamesList.map(mapGameToComponent) }
     </DataWraper>
   );
 };
 
+/** A component representing only a single game entry. */
+const SingleGame = (props: TGame) => {
+  const summary = useMemo(() => (props.summary).split(" ").splice(0, 35).join(" "), [props.summary])
+
+  return (
+  <GameArticle key={props.id}>
+    <Image></Image>
+
+    <TextWrapper>
+      <>
+        <Title>{props.name}</Title>
+
+        <Paragraph>{props.first_release_date}</Paragraph>
+        <Paragraph>{summary}</Paragraph>
+      </>
+      <Score>{props.rate}</Score>
+    </TextWrapper>
+  </GameArticle>);
+}
+
+/** Simply takes a game object and returns a component representing the entry. */
+const mapGameToComponent = (game: TGame) => <SingleGame {...game} />;
+
+type TRawGame = {
+  id: number;
+  first_relase_date: number;
+  name: string;
+  rating: number;
+  summary: string;
+}
+
+type TGame = {
+  id: number;
+  name: string;
+  rate: number;
+  summary: string;
+  first_release_date: Date;
+}
```

Comment:
That's some rather **meh** repetition, you can play around with the types if you like to make it less repetitive.